app/bundles/EmailBundle/Views/Email/list.html.php

```diff
@@ -217,6 +250,9 @@
                         <?php echo $view['content']->getCustomContent('email.stats', $mauticTemplateVars); ?>
                         <?php echo $view['content']->getCustomContent('email.stats.below', $mauticTemplateVars); ?>
                     </td>
+                    <td class="visible-lg"><?php echo $item->getDateAdded() ? $view['date']->toFull($item->getDateAdded()) : ''; ?></td>
```

Comment:
Better use toFullConcat method (https://github.com/mautic/mautic/blob/features/app/bundles/CoreBundle/Templating/Helper/DateHelper.php#L111) which make shorten date time without timezone on the end

Reply:
Yes, use toFullConcat is a better idea, i replaced toFull with it